tools/Azure.Mcp.Tools.Redis/src/RedisSetup.cs

```diff
@@ -13,6 +13,8 @@ public class RedisSetup : IAreaSetup
 {
     public string Name => "redis";
 
+    public string Title => "Azure Cache for Redis";
```

Comment:
```suggestion
    public string Title => "Azure Redis";
```
Reasoning: "Azure Cache for Redis" is a specific flavor of Azure Redis that is being retired. We're using "Azure Redis" as the umbrella term for all flavors including the new "Azure Managed Redis"